client/src/visualiser-src/graph-visualiser/util/codeSnippets.ts

```diff
@@ -0,0 +1,8 @@
+export const dfsCodeSnippet = `void dfs(int **M, int N, int src, bool *visited) {
+    visited[src] = true;
+    for (int v = 0; v < numVertices; v++) {
+        if (M[src][v] && !visited[v]) {
+            dfs(v);
```

Comment:
dfs() should be called with the correct amount of arguments

Reply:
Done.